### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by mehmeh1z** [tauraloke/tegownitsa#98] [Ideas]
Hi!
I like the idea of your app, but I think it lacks some features.
(maybe there is one, but I didn't find it.)
For example:
Adding tags to the name of the picture
The ability to sort the tagged pictures into folders that correspond to their tags.

**Comment by tauraloke** [comment]
Hello! Thank you for the feedback.

> Adding tags to the name of the picture

Do you mean adding tags to the filename in a hard disc for searching by the Explorer? I had a discussion around a same idea: adding tags to EXIF of an image for same possibility (btw it's not universal way for tag storing).
I think we can get overflow in filename length if tags count will be big enough. I have seen a filename tag limit around 5 or 10 in a tag system like Reactor.

> The ability to sort the tagged pictures into folders that correspond to their tags.

Tag system is not strict hierarchical. If you have folder system, you need organize duplicates of symlinks of your files to each tag folder.
I have problems with big folders so I organize files in the storage in the current order (Mediawiki-like). Yes, this is not the only way and may be it possible to organize a storage folder more human-friendly (as you suggest, for example).

**Comment by mehmeh1z** [comment]
**_Do you mean adding tags to the filename_**
Yep
**_I think we can get overflow in filename length if tags count will be big enough._**
You could make it so that you choose which tags are added to the file name (by series type, character type, etc.).
I.e. more global tags.
And make, for example, a setting so that you can pre-assign what tags will be written in the file name, so you do not have to choose each time yourself.
**_Tag system is not strict hierarchical. If you have folder system, you need organize duplicates of symlinks of your files to each tag folder._**
Well this problem is solved by what I described above.
